src/coaster_gui.cpp

```diff
@@ -428,9 +428,11 @@ void CoasterInstanceWindow::SetWidgetStringParameters(WidgetNumber wid_num) cons
 
 		case CIW_NUMBER_TRAINS:
 			_str_params.SetNumber(1, this->ci->number_of_trains);
+			_str_params.pluralize_amount = this->ci->number_of_trains;
 			break;
 		case CIW_NUMBER_CARS:
 			_str_params.SetNumber(1, this->ci->cars_per_train);
+			_str_params.pluralize_amount = this->ci->cars_per_train;
```

Comment:
Doesn't `Number` imply plural?  Or maybe add `SetCount` for adding plural? (or optional boolean whether plural is wanted)

Point is, most numbers will need a plural line, so you can either add many code lines for that, or hide it in the call.